model/src/test/kotlin/utils/FileArchiverTest.kt

```diff
@@ -60,10 +62,11 @@ class FileArchiverTest : StringSpec() {
         storage = FileProvenanceFileStorage(LocalFileStorage(storageDir), FileArchiverConfiguration.ARCHIVE_FILENAME)
     }
 
-    private fun createFile(path: String) {
+    private fun createFile(path: String): File {
```

Comment:
Note that this function always writes the path of the file to the file itself. So some parts of the file are always plain text. Which is a bit awkward if you want to test for (purely) binary files.